### DISCUSSION TITLE
使用 Vercel 部署，并启动了 Upstream Sync Action，自动更新版本时出现错误: Issue Close Require

### DISCUSSION THREAD

**Discussion by Nicecsh** [lobehub/lobehub#2619] [Self Hosting] [answered]
### 💻 系统环境

Windows

### 📦 部署环境

Vercel / Zeabur / Sealos

### 🌐 浏览器

Edge

### 🐛 问题描述

Fork了lobehub发布在Vercel ,并启动了 Upstream Sync Action，自动更新版本时出现下面的错误，不知道怎么解决。
![Screenshot_20240523_104458_com android chrome_edit_16252477912623](https://github.com/lobehub/lobe-chat/assets/22073409/8e4c9e24-b8d6-4920-8e24-af99c9cd162c)

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment, marked as answer]
这个没影响的，你可以直接删掉这几个文件

<img width="333" alt="image" src="https://github.com/lobehub/lobe-chat/assets/28616219/79471353-4579-4ba3-9e23-af43186306d5">

**Reply by Nicecsh** [reply]
删除这3个文件后，再没收到错误提示了，谢谢！👌🍵
